packages/measurement-value/float-cdf.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash');
+
+const { MeasurementValue } = require('./measurement-value');
```

Comment:
what would you think of this lib importing the ``cdf`` lib, then this class using the ``Cdf`` class underneath?

Reply:
It depends of code usage cases, but I believe that yes we can do it here.